src/olympia/stats/tests/test_cron.py

```diff
@@ -28,6 +28,23 @@ def test_stats_for_date(self):
         assert len(GlobalStat.objects.no_cache().filter(
             date=date, name=job)) == 1
 
+    def test_count_stats_for_date(self):
+        listed_addon = addon_factory()
+        unlisted_addon = addon_factory()
+        version_factory(
+            addon=listed_addon, channel=amo.RELEASE_CHANNEL_UNLISTED)
+        self.make_addon_unlisted(unlisted_addon)
```

Comment:
if you pass `version_kw={'channel': xxx}` into `addon_factory()` you don't need to make it unlisted a few lines further down.